Apps/Sandcastle/gallery/development/ScriptInInfoBox.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
You only need to include the file at `Apps/Sandcastle/gallery/ScriptInfoBox.html`, this second one is not needed.